src/pages/iou/request/step/IOURequestStepMerchant.tsx

```diff
@@ -123,10 +111,10 @@ function IOURequestStepMerchant({
             navigateBack();
             return;
         }
-        IOU.setMoneyRequestMerchant(transactionID, newMerchant, !isEditing);
+        IOU.setMoneyRequestMerchant(transactionID, newMerchant ?? '', !isEditing);
         if (isEditing) {
             // When creating new money requests newMerchant can be blank so we fall back on PARTIAL_TRANSACTION_MERCHANT
-            IOU.updateMoneyRequestMerchant(transactionID, reportID, newMerchant || CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT, policy, policyTags, policyCategories);
+            IOU.updateMoneyRequestMerchant(transactionID, reportID, newMerchant ?? CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT, policy, policyTags, policyCategories);
```

Comment:
```suggestion
            IOU.updateMoneyRequestMerchant(transactionID, reportID, newMerchant || CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT, policy, policyTags, policyCategories);
```

We will introduce a regression if we use `??` here